### PR TITLE
boards: shields: esp_8266: Add AT bin download link

### PR DESCRIPTION
**Pull request by nandojve**
Add Espressif At Bin download link page and small images adjusts.

Signed-off-by: Gerson Fernando Budke <nandojve@gmail.com>

### CODE REVIEW

File: boards/shields/esp_8266/doc/index.rst [hunk 1]
```diff
@@ -65,7 +65,7 @@ ESP8266 AT Bin 1.7.x available requires the following partition configuration
 to works:
 
 .. image:: ./esp_at_173.png
-   :width: 720px
+   :width: 678px
```
Comment (erwango) [inline boards/shields/esp_8266/doc/index.rst:68]: 813?

**Review by erwango — CHANGES_REQUESTED** [review]
Minor comment

File: boards/shields/esp_8266/doc/index.rst (hunk 1 above)
Comment (nandojve) [reply boards/shields/esp_8266/doc/index.rst:68]: I selected lower number than 720 to decrease a little bit the images. I'm complete open here, may you have any suggestion?

Comment (erwango) [reply boards/shields/esp_8266/doc/index.rst:68]: ok, I misunderstood your intention. That is fine

**Review by erwango — APPROVED** [review]

**Review by galak — APPROVED** [review]

**Pull request merged**